src/Cli/UpCommand.php

```diff
@@ -42,8 +41,7 @@ protected function configure()
     {
         $this
             ->setName('up')
-            ->setDescription('Start the project')
-        ;
```

Comment:
I think PSR-2 (or at least phpcs with that standard) doesn't like this. I prefer it the way it was though so undone my bad auto-formatting.